ert_gui/tools/load_results/load_results_tool.py

```diff
@@ -21,19 +21,20 @@
 
 
 class LoadResultsTool(Tool):
-    def __init__(self):
+    def __init__(self, facade):
+        self.facade = facade
         super(LoadResultsTool, self).__init__(
             "Load results manually",
             "tools/load_manually",
             resourceIcon("ide/table_import"),
         )
         self.__import_widget = None
         self.__dialog = None
-        self.setEnabled(LoadResultsModel.isValidRunPath())
+        self.setEnabled(LoadResultsModel.isValidRunPath(self.facade._enkf_main))
```

Comment:
accessing private member.

Reply:
It is not ideal, I know. If it is ok with your I would keep 2 of them, and make an issue to replace it completely with `LibresFacade`. The reason for keeping the 2 is then I dont have to drag `EnKFMain` deep into the GUI-tree